libs/common-const/src/cowprotocolTokenLists.ts

```diff
@@ -0,0 +1,6 @@
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+
+const tokenUrlRoot = 'https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images'
+
+export const cowprotocolTokenUrl = (address: string, chainId: SupportedChainId) =>
```

Comment:
is not a token url, it gives you the image, right?

Reply:
Also, the name of the file is a bit missleasing. This is not related to token lists, but to the images

Reply:
Yep, renamed to `cowprotocolTokenLogoUrl`, thanks!